src/components/actions/helpers.ts

```diff
@@ -32,7 +32,7 @@ export const swapOutAliasedAction = async (templateId: number, action: ActionInT
 
   const aliasedAction = await DBConnect.getSingleTemplateAction(templateId, code)
 
-  if (!aliasedAction) throw new Error('No Action matching alias')
+  if (!aliasedAction) throw new Error('No Action matching alias: ' + code)
```

Comment:
Unrelated, but small change to give more helpful error message for Preview errors

(See https://github.com/openmsupply/conforma-server/issues/1048#issuecomment-1581706398)